payment-manager/src/main/java/com/qpay/paymentmanager/model/dto/WalletModification.java

```diff
@@ -0,0 +1,25 @@
+package com.qpay.paymentmanager.model.dto;
+
+import com.qpay.libs.models.UserType;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+
+import java.math.BigDecimal;
+
+@Builder
+public record WalletModification (
```

Comment:
It is incorrect to use common entity for creation and deletion, because you need different fields during these 2 actions

Reply:
but i dont use walletModification for deletion

Reply:
I meant creation and modification